scripts/tests/user_mods_test3/shell_commands

```diff
@@ -1 +1,7 @@
-./xmlchange PIO_VERSION=2
\ No newline at end of file
+pioversion=`./xmlquery --value PIO_VERSION`
```

Comment:
Is this script to make sure that we test both versions of PIO (if the default is PIO1 test PIO2 and vice versa)?

Reply:
No, currently we are not testing pio2 - this script is to test an aspect of create_clone - (that you cannot use the same executable if you change an xml variable that has a build dependency).